.github/workflows/gating.yaml

```diff
@@ -104,17 +104,22 @@ jobs:
   build-image:
     name: Build Cachi2 image and run smoketest on it
```

Comment:
Since we're not running the smoketest anymore, we should rename this.

Reply:
renamed